envVariables.js

```diff
@@ -51,8 +50,8 @@ let enviromentVariables = {
 	},
 	SERVICE_NAME: {
 		message: 'current service name',
-		optional: true,
-		default: 'ml-projects-service',
+		optional: false,
+		default: '/project',
```

Comment:
@Prajwal17Tunerlabs if default value is given, we should keep it optional true, also if you are planning to use this for gotenberg callback, then default value should be "project-service"

Reply:
ok i will change the optional value
currently we are are using service-name as below in elevate-project
http://localhost:5000/**project**/v1/userProjects/......
so only ive kept it as "project"